src/main/java/uk/gov/hmcts/reform/bulkscan/orchestrator/services/MessageProcessor.java

```diff
@@ -22,47 +18,44 @@
     private static final Logger logger = LoggerFactory.getLogger(MessageProcessor.class);
 
     private final ReceiverProvider receiverProvider;
-    private final BulkScanProcessorClient bulkScanProcessorClient;
+    private final IMessageHandler envelopeProcessor;
 
-    public MessageProcessor(
-        ReceiverProvider receiverProvider,
-        BulkScanProcessorClient bulkScanProcessorClient
-    ) {
+    public MessageProcessor(ReceiverProvider receiverProvider, IMessageHandler envelopeProcessor) {
         this.receiverProvider = receiverProvider;
-        this.bulkScanProcessorClient = bulkScanProcessorClient;
+        this.envelopeProcessor = envelopeProcessor;
     }
 
     @Scheduled(fixedDelayString = "${queue.read-interval}")
     public void run() {
-        logger.info("Checking queue for new messages");
+        IMessage msg = null;
         try {
             IMessageReceiver msgReceiver = receiverProvider.get();
-            IMessage msg = msgReceiver.receive();
-            while (msg != null) {
-                handle(msgReceiver, msg);
-                msg = msgReceiver.receive();
+            while ((msg = msgReceiver.receive()) != null) {
+                process(msg);
+                msgReceiver.complete(msg.getLockToken());
             }
-        } catch (ServiceBusException | ConnectionException exc) {
```

Comment:
are those no longer thrown?

Reply:
they are but the action taken is the same. 
Adding the exception to the log (as is now) will provide the information needed.
We collapsed this due the the code coverage checks failing.

Since the action is the same it seems sensible to simplify the code.